src/controllers/handlers/about-handler.ts

```diff
@@ -55,31 +77,54 @@ export async function aboutHandler(
 
     const data: Record<string, string> = await response.json()
 
-    body.content.healthy = data['content'] === 'Healthy'
-    body.lambdas.healthy = data['lambda'] === 'Healthy'
-    body.comms.healthy = data['comms'] === 'Healthy'
+    result.content.healthy = data['content'] === 'Healthy'
+    result.lambdas.healthy = data['lambda'] === 'Healthy'
+    result.comms.healthy = data['comms'] === 'Healthy'
   } catch (err: any) {
     logger.error(err)
   }
 
-  if (commsProtocol === 'v3') {
+  const contentStatus = await status.getContentStatus()
+  if (contentStatus) {
+    const { version, commitHash } = contentStatus
+    result.content.version = version
+    result.content.commitHash = commitHash
+  }
+
+  const lambdasStatus = await status.getLambdasStatus()
+  if (lambdasStatus) {
+    const { version, commitHash } = lambdasStatus
+    result.lambdas.version = version
+    result.lambdas.commitHash = commitHash
+  }
+
+  let lighthouseRealmName: string | undefined = undefined
+  if (commsProtocol === 'v2') {
+    const lighthouseStatus = await status.getLighthouseStatus()
+    if (lighthouseStatus) {
+      const { version, commitHash, realmName } = lighthouseStatus
+      result.comms.version = version
+      result.comms.commitHash = commitHash
+      lighthouseRealmName = realmName
+    }
+  } else {
     const clusterStatus = await serviceDiscovery.getClusterStatus()
     if (clusterStatus.archipelago) {
-      body.comms = {
+      result.comms = {
+        ...result.comms,
         healthy: true,
         ...clusterStatus.archipelago
       }
     } else {
-      body.comms = {
-        healthy: false
-      }
+      result.comms.healthy = false
     }
   }
 
-  body.healthy = body.content.healthy && body.lambdas.healthy && body.comms.healthy
+  result.configurations.realmName = await realm.getName(lighthouseRealmName)
```

Comment:
`lighthouseRealmName` could be undefined, is this expected?

Reply:
yes, it's possible there is no lighthouse for example